sample/src/main/java/com/alamkanak/weekview/sample/MainActivity.java

```diff
@@ -133,8 +135,18 @@ public String interpretDate(Calendar date) {
             }
 
             @Override
-            public String interpretTime(int hour) {
-                return hour > 11 ? (hour - 12) + " PM" : (hour == 0 ? "12 AM" : hour + " AM");
+            public String interpretTime(int hour, int minutes) {
```

Comment:
Fixed the implementation of this in the sample so that it can actually show the half hours, if enabled.